website/www/site/content/en/get-started/quickstart/java.md

```diff
@@ -0,0 +1,185 @@
+---
+title: "Beam Quickstart for Java"
+aliases:
+  - /get-started/quickstart/
+  - /use/quickstart/
+  - /getting-started/
+---
+<!--
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Apache Beam Java SDK quickstart
+
+This quickstart shows you how to run an
+[example pipeline](https://github.com/apache/beam-starter-java) written with
+the [Apache Beam Java SDK](/documentation/sdks/java), using the
+[Direct Runner](/documentation/runners/direct/). The Direct Runner executes
+pipelines locally on your machine.
+
+If you're interested in contributing to the Apache Beam Java codebase, see the
+[Contribution Guide](/contribute).
+
+On this page:
+
+{{< toc >}}
+
+## Set up your development environment
+
+1. Download and install the
+  [Java Development Kit (JDK)](https://www.oracle.com/technetwork/java/javase/downloads/index.html)
```

Comment:
I copied this section from our existing quickstart. 

Qs:

- Is it better to be agnostic about how customers install the JDK and Gradle/Maven? My assumption was anyone using Java already has a preference. OTOH your approach gives customers definite steps to follow.
- If we show sdkman, will we have to keep the version # up-to-date?

Reply:
I've gone through the manual installs and they're all pretty bad. The instructions are not consistent and it's easy to get something wrong (like not setting JAVA_HOME). Maven specifically has pretty bad documentation on how to install, Gradle isn't as bad.

I think it's best to give an opinionated approach to install them, especially since it's _so much_ easier. We don't have to worry about keeping versions up to date, sdkman takes care of that. The only thing we have to make sure is that people install a compatible Java version (which I think it's currently only Java 11?).

If they're already using Java and already have a Java development environment set up, they can safely skip those steps. They're there mostly for people who don't have it ready. We can say something like "If you don't have Java installed, a simple way to install it is with `sdkman`", or something along those lines. What do you think?

Reply:
That makes sense. I rewrote this section to use sdkman. (It's definitely simpler!)